arbos/tx_processor.go

```diff
@@ -536,6 +536,11 @@ func (p *TxProcessor) EndTxHook(gasLeft uint64, success bool) {
 		posterFeeDestination = p.evm.Context.Coinbase
 	}
 	util.MintBalance(&posterFeeDestination, p.PosterFee, p.evm, scenario, purpose)
+	if p.state.ArbOSVersion() >= 10 {
+		if _, err := p.state.L1PricingState().AddToL1FeesAvailable(p.PosterFee); err != nil {
+			log.Error("failed to update L1FeesAvailable: "+err.Error(), "err", err)
```

Comment:
It seems like the error is repeated twice here. I'd recommend removing it from the main log message and just keeping it as a field.